coffee-module/coffee-module-mp/coffee-module-mp-restclient/pom.xml

```diff
@@ -20,6 +20,12 @@
 		<dependency>
 			<groupId>org.eclipse.microprofile.rest.client</groupId>
 			<artifactId>microprofile-rest-client-api</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>javax.inject</groupId>
+					<artifactId>javax.inject</artifactId>
```

Comment:
ezt sem latom hogy tartalmazna mar

Reply:
jogos, itt is volt verzióváltás mióta elkezdtem foglalkozni a feladattal. kiszedem.